src/connector/src/parser/unified/util.rs

```diff
@@ -15,13 +15,15 @@
 use risingwave_common::error::{ErrorCode, RwError};
 
 use super::{AccessError, ChangeEvent};
+use crate::parser::unified::ChangeEventOperation;
 use crate::parser::{SourceStreamChunkRowWriter, WriteGuard};
 
-pub fn apply_row_operation_on_stream_chunk_writer(
+pub fn apply_row_operation_on_stream_chunk_writer_with_op(
```

Comment:
I prefer to just split it to functions `write_upsert` and `wrtite_delete` to prevent any potential affect on the `non-upsert` source. But this PR is good enough now. c.c. @wugouzi